src/components/EventForm.jsx

```diff
@@ -24,14 +26,13 @@ const EventForm = ({ apiCall, ...reset }) => {
     errors,
     handleSubmit,
     touched,
-    resetForm,
     isSubmitting,
   } = useFormik({
     initialValues: {
-      title: "",
-      weddingDecription: "",
-      startTime: "",
-      endTime: "",
+      title: eventData?.title || "",
+      weddingDecription: eventData?.weddingDecription || "",
+      startTime: eventData?.startTime || "",
+      endTime: eventData?.endTime || "",
```

Comment:
enableReintialize property should be  added to integrate the api data in formik.

Reply:
Ahsan bai what is the purpose of enableReinitialize in our case because enableReinitialize is use when we want to extra things in the form after click on any things like dropdown and checkbox and want to more then one new field in form. so in our case nothing